lang/en.json

```diff
@@ -5,7 +5,7 @@
   "Pause": "Pause",
   "Replay": "Replay",
   "Current Time": "Current Time",
-  "Duration Time": "Duration Time",
```

Comment:
So, looking at the translations, most of them just translated 'Duration', so, I wonder whether we should just leave this as is? Or add Duration as a second option and then where we use it in the code we first try Duration localization and then Duration Time localization.

Reply:
"Duration Time" sounds like something poorly translated into English ;-)

Reply:
yup, makes sense, just thinking about how to reduce need for re-working all the translations.

Reply:
Could we just change it to "Duration" in this and all the other language files, and then note some of the translations may be incorrect? It seems like translations get fixed as much as they get added to?

Reply:
yeah, maybe that's just the best choice.

Reply:
Done.